### PR TITLE
Tomcat native openssl

### PR DESCRIPTION
**Pull request by amadigan**
I realize that Alpine has switched to LibreSSL (save for easy-rsa, also in community), but the upstream for this package does not yet support LibreSSL (https://bz.apache.org/bugzilla/show_bug.cgi?id=58434). The default Tomcat install logs an error (but does start) when the current tomcat-native is used. For various reasons it is useful to be able to run SSL in Tomcat, and I think it would be better to avoid having to choose between using tomcat-native and using SSL in Tomcat.

Sorry for submitting the PR twice, I realized after that tomcat-native did have tests, but there were some issues with the ant build.xml that executes them. I've added a patch to enable the tests, which do pass. I also tested manually with Tomcat 8.5.16 with SSL enabled.

### CODE REVIEW

File: community/tomcat-native/APKBUILD [hunk 1]
```diff
@@ -1,45 +1,55 @@
...
-pkgrel=0
+pkgrel=1
 pkgdesc="Native resources optional component for Apache Tomcat"
 url="http://tomcat.apache.org/native-doc/"
 arch="all"
 license="ASL-2.0"
-depends="openjdk8-jre-base"
-makedepends="apr-dev chrpath openjdk8"
+depends="openjdk8-jre-base openssl"
+makedepends="apr-dev chrpath openjdk8 openssl-dev apache-ant"
```
Comment (jirutka) [inline community/tomcat-native/APKBUILD]: `apache-ant` should be in `checkdepends`.

File: community/tomcat-native/APKBUILD [hunk 2]
```diff
@@ -1,45 +1,55 @@
...
 subpackages="$pkgname-dev"
-source="http://www-eu.apache.org/dist/tomcat/tomcat-connectors/native/$pkgver/source/$pkgname-$pkgver-src.tar.gz"
-builddir="$srcdir/$pkgname-$pkgver-src/native"
+source="http://www-eu.apache.org/dist/tomcat/tomcat-connectors/native/$pkgver/source/$pkgname-$pkgver-src.tar.gz enable-tests.patch"
```
Comment (jirutka) [inline community/tomcat-native/APKBUILD]: Please put each source on a separate line.

**Comment by jirutka** [comment]
I’d prefer to split this package into two variants – tomcat-native w/o TLS and tomcat-native-tls with TLS, each with `depends="!<other>"` and `replaces="<other>"` (`<other>` means `$pkgname` or `$pkgname-tls`).

**Comment by amadigan** [comment]
I'll update with the fixes and split this to a "tomcat-native-openssl" package (is that name OK?). I'll also see if I can get the tests to pass without SSL.

**Comment by amadigan** [comment]
Should I close the request?

**Pull request closed without merging**